packages/cashier/src/Containers/deposit.jsx

```diff
@@ -42,6 +45,13 @@ const Deposit = ({
     setSideNotes,
     tab_index,
 }) => {
+    const is_fiat_currency_banner_visible_for_MF_clients =
+        !is_switching &&
+        landing_company_shortcode === 'maltainvest' &&
+        !is_crypto &&
+        !can_change_fiat_currency &&
+        !!iframe_height;
```

Comment:
why do we need to check `is_switching` and `iframe_height` here?

Reply:
@farrah-deriv 
1) regarding the iframe_height when the MF client landed on cashier/deposit page he will see side note on the right side of the screen, then as cashier-premier iframe is not loaded, this note will move to the left to fill an empty space of the screen and after the iframe loaded, side note again will move to the right, so to avoid this movements of the side note i decided to show cashier-premier iframe and side note at the same time (to improve user experience).
2) regarding is_switching property i have checked it again and in this case it is not needed (i will remove it)